portfolio/src/main/webapp/script.js

```diff
@@ -54,22 +54,50 @@ function addRandomTVQuote() {
 }
 
 /**
- * Using async and await to fetch list data.
+ * Load comments from server and add them to page.
  */
-async function getDataWithAsyncAwait() {
-  const response = await fetch('/data');
-  const messages = await response.json();
+function loadComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
```

Comment:
When you're chaining lots of calls together, but at different levels like this it's hard easily parse it, try separating the .then calls onto different lines eg:
```
// line up second then with first then
fetch('/data').then(response => response.json())
              .then((comments) => {
```
or
```
// indent both thens by 4 instead of two because they're a continuation of the same line
fetch('/data')
    .then(response => response.json())
    .then((comments) => {
```

Reply:
I tried to do this. I got a bit confused by the bracketing, not sure if I did it right?

Reply:
yeah looks good I think, honestly I'm used to the autoformatter doing this